INSTALL.md

```diff
@@ -159,10 +159,8 @@ del /F /Q "%KRATOS_BUILD%\%KRATOS_BUILD_TYPE%\CMakeFiles"
 
 rem Configure
 @echo on
-cmake -G"Visual Studio 16 2019" -H"%KRATOS_SOURCE%" -B"%KRATOS_BUILD%\%KRATOS_BUILD_TYPE%"          ^
--DINCLUDE_FEAST=OFF                                                                                 ^
--DLAPACK_LIBRARIES="C:\CompiledLibs\blas_x64\liblapack.lib"                                         ^
```

Comment:
This information should probably be moved to the README of the ExternalSolversApplication, otherwise it will become quite hard to figure out how to compile it.

Reply:
Good point!

Reply:
done